supply/lib/supply/uploader.rb

```diff
@@ -230,14 +355,29 @@ def update_obb(apk_version_code, expansion_file_type, references_version, file_s
     end
 
     def update_track(apk_version_codes)
+      return if apk_version_codes.empty?
+
       UI.message("Updating track '#{Supply.config[:track]}'...")
       check_superseded_tracks(apk_version_codes) if Supply.config[:check_superseded_tracks]
```

Comment:
@joshdholtz TEST THIS PLEASE

Reply:
Deleted this because Google Play does this itself now 
![Screen Shot 2019-11-06 at 7 05 31 AM](https://user-images.githubusercontent.com/401294/68301617-f7f5cf00-0065-11ea-8220-2ada7e826262.png)